DatadogInternal/Tests/NetworkInstrumentation/NetworkInstrumentationFeatureTests.swift

```diff
@@ -466,101 +466,23 @@ class NetworkInstrumentationFeatureTests: XCTestCase {
         )
     }
 
-    // MARK: - URLRequest Interception
+    // MARK: - URLSessionTask Interception
 
-    func testGivenOpenTracing_whenInterceptingRequests_itInjectsTrace() throws {
-        // Given
```

Comment:
💡 All these tests are not necessary because we removed `traceContext()` in `DatadogURLSessionHandler`.